Externals/librocket/Source/Core/ElementDocument.cpp

```diff
@@ -129,11 +129,14 @@ void ElementDocument::ProcessHeader(const DocumentHeader* document_header) {
 
   // Load external scripts.
   for (size_t i = 0; i < header.scripts_external.size(); i++) {
-    StreamFile* stream = new StreamFile(m_context);
-    if (stream->Open(header.scripts_external[i]))
-      LoadScript(stream, header.scripts_external[i]);
-
-    stream->RemoveReference();
+    if (m_context->LoadScriptContents()) {
+      StreamFile* stream = new StreamFile(m_context);
+      if (stream->Open(header.scripts_external[i]))
```

Comment:
Missing explicit `{}`

Reply:
ooo thanks! 
I think librocket typically ignores the {} so thats where this crept in, but as I go forward and make changes, I will try to keep it MORE like our coding style.